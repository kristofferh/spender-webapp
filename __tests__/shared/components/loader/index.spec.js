import React from "react";
import renderer from "react-test-renderer";

import Loader from "shared/components/loader";

describe("Create Loader snapshot", () => {
  it("should render and match the snapshot", () => {
    const tree = renderer.create(<Loader size={30} color="#ee99ee" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
