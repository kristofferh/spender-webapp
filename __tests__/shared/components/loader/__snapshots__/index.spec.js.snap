// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Create Loader snapshot should render and match the snapshot 1`] = `
.emotion-4 {
  vertical-align: middle;
  position: relative;
}

.emotion-2 {
  margin: 10px auto;
  border: 6px solid #dedede;
  border-radius: 50%;
  width: 30px;
  height: 30px;
}

.emotion-0 {
  -webkit-animation: animation-6bj4oy-buttonSpin 0.9s infinite cubic-bezier(0.08,0.03,1,1.04);
  animation: animation-6bj4oy-buttonSpin 0.9s infinite cubic-bezier(0.08,0.03,1,1.04);
  display: block;
  position: absolute;
  -webkit-transform: translate(-50%,-50%);
  -ms-transform: translate(-50%,-50%);
  transform: translate(-50%,-50%);
  pointer-events: none;
  left: 50%;
  top: 50%;
}

<div
  className="emotion-4 emotion-5"
>
  <div
    className="emotion-2 emotion-3"
  >
    <svg
      className="emotion-0 emotion-1"
      height={30}
      viewBox="0 0 41 41"
      width={30}
    >
      <path
        d="M38,20.5 C38,30.1685093 30.1685093,38 20.5,38"
        fill="none"
        stroke="#ee99ee"
        strokeWidth={6}
      />
    </svg>
  </div>
</div>
`;
